content/monitors/monitor_types/anomaly.md

```diff
@@ -167,10 +167,18 @@ We used to expose an algorithm called `adaptive` which would try to figure out a
 
 Previously we were treating count metrics as gauges, and thus interpolating between reported points. This led to some very odd-looking metrics for sparsely reported counts. Anomalies are no longer interpolating between counts, but for legacy monitors, the old behavior is preserved using the `count_default_zero` argument.
 
+### But what if I prefer it if my count metric were treated as a gauge?
+
+Not interpolating between counts makes sense if the thing you are counting is something like errors. However, if you have regularly scheduled jobs that happen every hour, it might make more sense if the metric is not reporting a value of 0.0 between runs. There are two different ways to accomplish this: 1) set the rollup to be one hour; or 2) explicitly set `count_default_zero='false'` using the API.
```

Comment:
nit: `set the rollup to be one hour` -> `set the rollup interval (found in the advanced options section) to be one hour`

Is there a page in the docs for anomalies (not anomaly monitors) where we should address this same topic but for dashboards? I think the support ticket that came in related to this was for a screenboard.

Reply:
I can't seem to find a non-monitor anomalies page in the docs site, so I guess not?

Reply:
yeah, when the docs site got refactored anomalies was moved under alerting (so i stripped out the dashboard stuff)